meson.build

```
@@ -0,0 +1,11 @@
+# This is just a stub project to include all the others as subprojects
+# for development shell purposes
+
+project('nix-dev-shell', 'cpp',
+  version : run_command('cat', './.version', check : true).stdout().strip(),
```

What's the minimum version of meson you support? This doesn't list `meson_version: '>= 1.1'` -- you're using `meson.options` which requires this.

The perl subproject requires 0.64.0, and the earliest version of meson in repology for nixos is 0.57.1

Meson 0.57.0 added support for this:

```suggestion
  version : files('.version'),
```

upon which meson itself will read this file for you without forking out to `cat` (which in theory may not be in $PATH, and e.g. on Windows may not exist at all absent msys2 or cygwin).

---

Note there are a couple other files that also use the `cat` approach, I just only reviewed the first location. :)